build-tools/timing/timing.targets

```diff
@@ -0,0 +1,148 @@
+        ToolPath="$(GitToolPath)"
+        ToolExe="$(GitToolExe)" />
+    <Exec Command="$(_XABuild) /fl /flp:LogFile=$(_OutputFile).log &quot;/logger:$(_TimingLogger);ID=$(_ID);Description=$(_Description);Commit=$(XAVersionHash);OutputPath=$(_OutputFile).xml&quot; /p:Configuration=$(Configuration) %(XACaptureBuildTimingProject.Identity) /t:$(_Target)" />
+  </Target>
+  <Target Name="FreshInstall"
+      Outputs="$(_TopDir)bin\Test$(Configuration)\Timing_FreshInstall_$(Configuration)_%(XACaptureBuildTimingProject.Name).xml">
+    <PropertyGroup>
+      <_ID>@(XACaptureBuildTimingProject->'%(ShortName)')-FreshInstall</_ID>
+      <_Description>An install after a fresh build for @(XACaptureBuildTimingProject->'%(Name)')</_Description>
+      <_Target>Install</_Target>
+      <_OutputFile>$(_OutputDir)Timing_$(_ID)_$(Configuration)_@(XACaptureBuildTimingProject->'%(Name)')</_OutputFile>
+    </PropertyGroup>
+    <Exec Command="$(_XABuild) /fl /flp:LogFile=$(_OutputFile).log &quot;/logger:$(_TimingLogger);ID=$(_ID);Description=$(_Description);Commit=$(XAVersionHash);OutputPath=$(_OutputFile).xml&quot; /p:Configuration=$(Configuration) %(XACaptureBuildTimingProject.Identity) /t:$(_Target)" />
+  </Target>
+  <Target Name="SecondBuild"
+      Outputs="$(_TopDir)bin\Test$(Configuration)\Timing_SecondBuild_$(Configuration)_%(XACaptureBuildTimingProject.Name).xml">
+    <PropertyGroup>
+      <_ID>@(XACaptureBuildTimingProject->'%(ShortName)')-SecondBuild</_ID>
+      <_Description>A second build after a fresh build for @(XACaptureBuildTimingProject->'%(Name)')</_Description>
+      <_Target>SignAndroidPackage</_Target>
+      <_OutputFile>$(_OutputDir)Timing_$(_ID)_$(Configuration)_@(XACaptureBuildTimingProject->'%(Name)')</_OutputFile>
+    </PropertyGroup>
+    <Exec Command="$(_XABuild) /fl /flp:LogFile=$(_OutputFile).log &quot;/logger:$(_TimingLogger);ID=$(_ID);Description=$(_Description);Commit=$(XAVersionHash);OutputPath=$(_OutputFile).xml&quot; /p:Configuration=$(Configuration) %(XACaptureBuildTimingProject.Identity) /t:$(_Target)" />
+  </Target>
+  <Target Name="SecondInstall"
+      Outputs="$(_TopDir)bin\Test$(Configuration)\Timing_SecondInstall_$(Configuration)_%(XACaptureBuildTimingProject.Name).xml">
+    <PropertyGroup>
+      <_ID>@(XACaptureBuildTimingProject->'%(ShortName)')-SecondInstall</_ID>
+      <_Description>An install after a second build for @(XACaptureBuildTimingProject->'%(Name)')</_Description>
+      <_Target>Install</_Target>
+      <_OutputFile>$(_OutputDir)Timing_$(_ID)_$(Configuration)_@(XACaptureBuildTimingProject->'%(Name)')</_OutputFile>
+    </PropertyGroup>
+    <Exec Command="$(_XABuild) /fl /flp:LogFile=$(_OutputFile).log &quot;/logger:$(_TimingLogger);ID=$(_ID);Description=$(_Description);Commit=$(XAVersionHash);OutputPath=$(_OutputFile).xml&quot; /p:Configuration=$(Configuration) %(XACaptureBuildTimingProject.Identity) /t:$(_Target)" />
+  </Target>
+  <Target Name="TouchCSharpBuild"
```

Comment:
I see a target `TouchCSharpBuild`. Should the timing collected by this target be mentioned in the commit message?